### DISCUSSION TITLE
which Dremel 3D20 Firmware i have to use to install Marlin?

### DISCUSSION THREAD

**Discussion by sheshin02** [moonglow/FlashForge_Marlin#133] [Q&A] [answered]
i was reading the [instructions to install Marlin ](https://github.com/moonglow/FlashForge_Marlin/wiki/Installing-Marlin-on-Dremel-3D20) in the dremel printer and when i got to the point that reads " Copy the Marlin firmware that has been encrypted with the Dremel3D20 encryption key " i was stuck because looks like there are two versions of the dremel firmware and im not sure which i have to use

dremel_2.0.9.5_10022022.bin
dremel_2.0.9.5_la_10022022.bin

i could not find what makes them different and i dont want to risk using the wrong one so if anyone knows which should i use please let me know

**Comment by tracedgod** [comment, marked as answer]
Hi there, 

Either firmware builds will work, the ``dremel_2.0.9.5_la_10022022.bin`` is the firmware using the ``linear advance`` option for marlin. If you do not know what it does, then I do not recommend flashing it. You can get some info about which builds do what [here.](https://github.com/moonglow/FlashForge_Marlin/wiki/Prebuilt-Marlin-firmware)
